templates/room.hbs

```diff
@@ -119,6 +119,7 @@
                 personal-space-bubble
                 look-controls
                 networked="template: #head-template; showLocalTemplate: false;" 
+                spawn-controller="radius: 4"
```

Comment:
Hmm this should not be on the head, it should be on the rig, that does mean we will have to deal with moving the rig with respect to the head, like we do for teleporting. Otherwise the rig is going to remain at the spawn point and also this position is going to be wiped out by the HMD